### DISCUSSION TITLE
How to use it with multiple databases in a multi tenant app?

### DISCUSSION THREAD

**Discussion by fabianpnke** [spatie/db-dumper#210] [General]
I'm running a multi tenant app with multiple databases. (spatie/laravel-multitenancy)
How can I backup each database, since the tenant connection uses "null" as the db_name?

**Comment by aveein** [comment]
you can use following for backing up all database.
 MySql::create()
    ->setUserName('username')
    ->setPassword('password')
    ->addExtraOption('--all-databases')
    ->getDumpCommand('dump.sql', 'credentials.txt');
    
 For single tenant you can pass your tenant database
